config/test.exs

```diff
@@ -1,34 +1,9 @@
-# This file is responsible for configuring your application
-# and its dependencies with the aid of the Mix.Config module.
-use Mix.Config
+import Config
 
-config :streamex, region: "",
-                  key: "8andj8c67ycp",
-                  secret: "6anbcfx3a5awa62ehtmjfnzdn74shg9tcbbwrpfrdmeaf2t5f3pddfsj9ewpjnv7"
+config :streamex,
+  region: "",
+  key: "bxfd3qv7c56c",
+  secret: "8zmdu5aq6v6uk273ht9jzsnqnh8v33dsp9f23pjh3ngegc9pndeg2ntgk594quee"
```

Comment:
If this is okay for now (no worse than before), I can do a follow-up to remove the secrets from config and VCRs.